README.md

```diff
@@ -16,3 +16,39 @@ Check out our [getting started Guide](https://developers.livestorm.co/docs/getti
 ### You need a hand or want to contribute?
 
 We are here to help, feel free to drop an issue to our [Github repo](https://github.com/livestorm/livestorm-plugin), or to contact us directly at [plugins@livestorm.co](mailto:plugins@livestorm.co) 
+
+### How to run E2E tests
+
+You are contributing and you want to run the E2E tests before pushing your commit ?You have only have to make the followings steps:
+1. Set your Livestorm credentials in a cypress.env.json file
+
+    ```json
+    {
+
+        "TEAM_MEMBER_EMAIL": "email@livestorm.co", // The email of a team member of your organization
+
+        "TEAM_MEMBER_PASSWORD": "password" // The password of the latter
+    }
+    ```
+
+
+2. Set your plugin organization settings in a plugin/environments.json
+
+    ```json
+    {
+
+        "development": {
+
+            "apiKey": "${apiKey}", // The API Key generated on your organization
+
+            "name": "plugin-name", // The unique name of the plugin published to your organization
```

Comment:
I think this should be a dedicated file "CONTRIBUTING.md" with a small introduction : 
```
### You need a hand or want to contribute?

We are here to help, feel free to drop an issue to our [Github repo](https://github.com/livestorm/livestorm-plugin), or to contact us directly at [plugins@livestorm.co](mailto:plugins@livestorm.co)

## Running E2E

...
```

Reply:
See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Reply:
Thanks guys. I will take a look.